### PR TITLE
Add unbaffeld and inference-tools recipes

### PR DESCRIPTION
**Pull request by JarrodLeddy**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JarrodLeddy** [comment]
@conda-forge/help-python package has passed tests and is ready for review

File: recipes/unbaffeld/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "unbaffeld" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/c4/a3/3dc6f9912137aab7da98640f90ac7ab314128c5c417d4f1b6321196ca0e5/UNBAFFELD-0.2.2.tar.gz"
```
Comment (synapticarbors) [inline recipes/unbaffeld/meta.yaml:9]: It's probably better to use the canonical url that we have in the example:

```
url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/unbaffeld/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```
Comment (synapticarbors) [inline recipes/unbaffeld/meta.yaml:21]: I'm seeing other run time requirements in the `setup.py`:

```
requirements = [
    "inference-tools>=0.12.0",
    "numpy>=1.8",
    "scipy>=1.6.3",
    "h5py",
    "matplotlib>=3.4.2"
]
```

NOTE: you probably want to make the dependency `matplotlib-base` unless you have need for the QT gui.

File: recipes/unbaffeld/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - python >=3.7
+
+test:
```
Comment (synapticarbors) [inline recipes/unbaffeld/meta.yaml:24]: I would recommend adding a `pip check` test. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

File: recipes/unbaffeld/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  run:
+    - python >=3.7
+    - inference-tools >=0.12.0
+    - numpy >=1.8
+    - scipy >=1.6.3
+    - h5py
+    - matplotlib >=3.4.2
+
+test:
+  imports:
```
Comment (synapticarbors) [inline recipes/unbaffeld/meta.yaml:31]: You only need to import this package, not it's dependencies

Comment (JarrodLeddy) [reply recipes/unbaffeld/meta.yaml:30]: done

File: recipes/unbaffeld/meta.yaml (hunk 1 above)
Comment (JarrodLeddy) [reply recipes/unbaffeld/meta.yaml]: I had trouble with the link but realized it was a capitalization issue with the example one above. Have modified it accordingly.

File: recipes/unbaffeld/meta.yaml (hunk 2 above)
Comment (JarrodLeddy) [reply recipes/unbaffeld/meta.yaml:21]: Done.

File: recipes/unbaffeld/meta.yaml (hunk 3 above)
Comment (JarrodLeddy) [reply recipes/unbaffeld/meta.yaml:29]: Done.

**Comment by JarrodLeddy** [comment]
I'm seeing inference tools unable to be found.  It's only up on pypi, not conda forge, so I'm guessing I'll need to get that onto conda forge before I can get my package up.

**Comment by synapticarbors** [comment]
You can open a separate PR for `inference-tools` or include it in this PR. See:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#adding-multiple-packages-at-once

**Comment by synapticarbors** [comment]
I think this has something to do with missing `setuptools_scm`, which I think sets the internal version? Try adding `setuptools_scm` and `setuptools` to the `host` section. Also check to see if that's needed for the version of `unbaffeld` as well.

**Pull request merged**